docs/sources/flow/reference/components/module.http.md

```diff
@@ -0,0 +1,165 @@
+---
+title: module.http
+labels:
+  stage: beta
+---
+
+# module.http
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`module.http` is a *module loader* component. A module loader is a Grafana Agent Flow
+component which retrieves a [module][] and runs the components defined inside of it.
```

Comment:
```suggestion
`module.http` is a *module loader* component available with Grafana Agent in flow mode. 
The loader retrieves a [module][] and runs the components defined in the module.
```

Could we rephrase this a little? Does this lose the meaning and intent of the definition?

Reply:
The second sentence sounds good to me :smile:. For the first, I realized now there is a [Module Loader section in our existing docs](https://grafana.com/docs/agent/latest/flow/concepts/modules/#module-loaders) that I think makes more sense to just link to, rather than re-explain.

Reply:
I agree. Linking to the Module Loader section is better. I'll get to editing that existing doc at some point as well :-)